modules/generator/instance.go

```diff
@@ -113,60 +116,92 @@ func (i *instance) watchOverrides() {
 	}
 }
 
-func (i *instance) updateProcessors(desiredProcessors map[string]struct{}) error {
+func (i *instance) updateProcessors() error {
+	desiredProcessors := i.overrides.MetricsGeneratorProcessors(i.instanceID)
```

Comment:
should this use some kind of default value if an instance has no overrides? are overrides required to turn on processors for a tenant?